src/App.tsx

```diff
@@ -15,37 +16,85 @@
   'Garlic',
 ];
 
+enum Way {
+  'Unknown' = 0,
+  'Info' = 'is-info',
+  'Success' = 'is-success',
+}
+
+function getSorted(obj: string[], way: Way) {
+  const currentGoods = [...obj];
+
+  currentGoods.sort((el1, el2) => {
+    switch (way) {
+      case 'is-info':
+        return el1.localeCompare(el2);
+      case 'is-success':
+        return el1.length - el2.length;
+      default:
+        return 0;
+    }
+  });
+
+  return currentGoods;
+}
+
 export const App: React.FC = () => {
+  const [selectItem, setSelectItem] = useState('');
+  const [selReverse, setselReverse] = useState('');
+
+  const goods = getSorted(goodsFromServer, selectItem as Way);
```

Comment:
Casting `selectItem` to `Way` might lead to runtime errors if `selectItem` is not a valid `Way` value. Ensure that `selectItem` is always a valid `Way` value before passing it to `getSorted`.